Composer/packages/extensions/adaptive-form/src/components/AdaptiveForm/index.tsx

```diff
@@ -87,7 +87,7 @@ export const AdaptiveForm: React.FC<AdaptiveFormProps> = function AdaptiveForm(p
   }, [currentDialog, focusPath, localData]);
 
   if (!localData) {
-    return <>No Data</>;
+    return null;
```

Comment:
Why this? Should I add a loading indicator?

Reply:
Yeah, let's either style this a bit or add a proper loading affordance

Reply:
I'll cut a ticket